Source/WebCore/page/scrolling/ScrollingTree.h

```diff
@@ -141,6 +141,7 @@ class ScrollingTree : public ThreadSafeRefCounted<ScrollingTree> {
 #endif
     virtual void scrollingTreeNodeWillStartScroll(ScrollingNodeID) { }
     virtual void scrollingTreeNodeDidEndScroll(ScrollingNodeID) { }
+    virtual void scrollingTreeNodeWillCancelScroll() { }
```

Comment:
This naming isn't right, since this action isn't associated with a specific `ScrollingNodeID`; it's not an analogue of `scrollingTreeNodeDidEndScroll()`.

Why not just make `clearNodesWithUserScrollInProgress()` virtual, and override in `RemoteScrollingTree`?

Reply:
Will do